src/main/java/teammates/ui/template/FeedbackQuestionFeedbackPathSettings.java

```diff
@@ -48,4 +55,20 @@ public void setNumOfEntitiesToGiveFeedbackToValue(int numOfEntitiesToGiveFeedbac
         this.numOfEntitiesToGiveFeedbackToValue = numOfEntitiesToGiveFeedbackToValue;
     }
     
+    public String getCustomFeedbackPathsSpreadsheetData() {
+        return customFeedbackPathsSpreadsheetData;
+    }
+
+    public void setCustomFeedbackPathsSpreadsheetData(List<FeedbackPathAttributes> feedbackPathAttributesList) {
+        List<List<String>> customFeedbackPaths = new ArrayList<List<String>>();
+        for (FeedbackPathAttributes feedbackPathAttributes : feedbackPathAttributesList) {
+            customFeedbackPaths.add(Arrays.asList(feedbackPathAttributes.getGiver(),
+                                                  feedbackPathAttributes.getRecipient()));
+        }
+        List<String> customFeedbackPathStrings = new ArrayList<String>();
+        for (List<String> customFeedbackPath : customFeedbackPaths) {
+            customFeedbackPathStrings.add(Sanitizer.sanitizeListForCsv(customFeedbackPath).toString());
```

Comment:
I added a custom path for a team containing `"` in the name, and the spreadsheet can't load anymore